.studio/automate-gateway

```diff
@@ -554,21 +554,19 @@ function get_api_token_with_policy() {
   check_if_deployinate_started || return 1
 
   local policy=${1:?"usage: get_api_token_with_policy POLICYNAME"}
-  local iam_version
-  iam_version="$(chef-automate iam version | awk '{print $2}')"
   local token_description="api-token-$$-${policy}"
-  local token_file="/tmp/api_token_${iam_version}_${policy}"
+  local token_file="/tmp/api_token_${policy}"
 
   if [ -f "$token_file" ]; then
     cat "$token_file"
   else
     # create standard api token
-    chef-automate iam token create "$token_description" >"$token_file" || return 1
+    chef-automate iam token create "$token_name" >"$token_file" || return 1
     # retrieve the tokens' ID
     local token_id
     token_id="$(curl -fsS --insecure -H "api-token: $(get_admin_token)" \
-      "${GATEWAY_URL}/auth/tokens" |\
-        jq -er --arg token "$token_description" '[.tokens[] | select(.description == $token) | .id] | first' )"
+      "${GATEWAY_URL}/iam/v2/tokens" |\
```

Comment:
do we want the name here or the id? finding the jq a little hard to read 🤔

Reply:
I think we still want the id (to be added to the policy below)